### PR TITLE
PMM-7329 hide checks table if STT is disabled

### PR DESCRIPTION
**Pull request by nicolalamacchia**
https://github.com/Percona-Lab/pmm-submodules/pull/1451

### CODE REVIEW

File: pmm-app/src/pmm-check/CheckPanel.tsx [hunk 1]
```diff
@@ -76,7 +79,14 @@ export const CheckPanel: FC = () => {
             {tabs.map((tab) => tab.key === activeTab && tab.component)}
           </TabContent>
         </>
-      )}
+      ) : (
+        <div className={styles.empty} data-qa="db-check-panel-settings-link">
+          Security Threat Tool is disabled. You can enable it in&nbsp;
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-check/CheckPanel.tsx:84]: I think this part of the message can go to the Messages file.

File: pmm-app/src/pmm-check/CheckPanel.tsx [hunk 2]
```diff
@@ -76,7 +79,14 @@ export const CheckPanel: FC = () => {
...
+          <Link className={styles.link} to={PMM_SETTINGS_URL}>
+            PMM Settings.
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-check/CheckPanel.tsx:86]: And this too. As two separate strings.

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: pmm-app/src/pmm-check/CheckPanel.test.tsx [hunk 3]
```diff
@@ -58,4 +58,27 @@ describe('CheckPanel::', () => {
...
+      },
+    }));
+
+    const wrapper: ReactWrapper<{}, {}, any> = mount(<CheckPanelRouter />);
+
+    await runAllPromises();
+    wrapper.update();
+
+    expect(wrapper.find(dataQa('db-check-panel-settings-link'))).toHaveLength(1);
+    const text = 'Security Threat Tool is disabled. You can enable it in PMM Settings.';
```
Comment (fabio-silva) [inline pmm-app/src/pmm-check/CheckPanel.test.tsx:75]: Probably this should be stored in Messages and used both here and in the component

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: pmm-app/src/pmm-check/CheckPanel.test.tsx (hunk 3 above)
Comment (nicolalamacchia) [reply pmm-app/src/pmm-check/CheckPanel.test.tsx:75]: Done [in this PR](https://github.com/percona/grafana-dashboards/pull/1109)

File: pmm-app/src/pmm-check/CheckPanel.tsx (hunk 2 above)
Comment (nicolalamacchia) [reply pmm-app/src/pmm-check/CheckPanel.tsx:86]: Done [in this PR](https://github.com/percona/grafana-dashboards/pull/1109)

File: pmm-app/src/pmm-check/CheckPanel.tsx (hunk 1 above)
Comment (nicolalamacchia) [reply pmm-app/src/pmm-check/CheckPanel.tsx:84]: Done [in this PR](https://github.com/percona/grafana-dashboards/pull/1109)

**Review by tiagomotasantos — APPROVED** [review]

**Review by fabio-silva — APPROVED** [review]

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**